mbed_build/_internal/find_files.py

```diff
@@ -51,3 +89,10 @@ def _matches_mbedignore_patterns(path: Path, patterns: Iterable[str]) -> bool:
     """Check if given path matches one of the .mbedignore patterns."""
     stringified = str(path)
     return any(fnmatch(stringified, pattern) for pattern in patterns)
+
+
+def _matches_label_rules(path: Path, label_type: str, allowed_label_values: set) -> bool:
+    """Check if given path contains only allowed values for a given label type."""
```

Comment:
could you put an example in the docs?

Reply:
https://github.com/ARMmbed/mbed-build/pull/7/commits/7847af4ef77ad53e1cca8ba1d73587dd4a9d6a66 <- I've removed this function and improved the docs where it was previously used.